gradle/libs.versions.toml

```diff
@@ -5,6 +5,7 @@ format.version = "1.1"
 assertj = "3.24.2"
 awaitility = "4.2.0"
 edc = "0.3.2-SNAPSHOT"
+identityhub = "0.3.1"
```

Comment:
maybe this deserves a comment, I guess in the future the goal will be to align the versions again